spec/interfaceJasmineSpec.js

```diff
@@ -0,0 +1,143 @@
+describe("Interface", function() {
+
+  // let dummyRequest = function(theFunction, stub = stubbedData) {
+  //   return stubbedData
+  // }
+
+  beforeEach(function() {
+    let testDiv = document.createElement("div");
+    testDiv.setAttribute('id', 'cryer-div');
+    document.body.appendChild(testDiv)
+    document.body.insertAdjacentHTML('beforeend',
+      `<div id="welcome-wagon-div">
+        <h3 id="welcome-wagon-header">Welcome Guest Cryer!</h3>
+        <h4 id="welcome-wagon-subheader">Without signing in you can look but you can't cry!</h4>
+        <div id="signup-Login-Div">
+        <button type="button" id="sign-up">Sign Up!</button>
+        <button type="button" id="log-in">Log In!</button>
+      </div>
+      <form id="signup-form">
+        <input type="text" placeholder="username" required>
+        <input type="text" placeholder="password" id="password-entry" required>
+        <input type="submit" value="Submit">
+      </form>
+      <form id="login-form">
+        <input type="text" placeholder="username" required>
+        <input type="text" placeholder="password" id="password-entry" required>
+        <input type="submit" value="Submit">
+      </form>
+      <form id="cry-entry-box" hidden="true">
+        <textarea id='cryArea' rows="15" cols="42" placeholder="Write your Cry here!"></textarea>
+        <input type="submit" value="Submit">
+      </form>
+      </div>`
+    )
+  })
```

Comment:
Good setup for your tests, was this written before the page was set up? à la TDD?